app/views/assignments/list_submissions.html.erb

```diff
@@ -55,13 +55,13 @@
  			
  			<% @participants.each do |participant| %>
     			<%= "Hyperlinks:<br/>".html_safe if !participant.hyperlinks.blank? and participant == @participants.first %>
-	    		<% participant.hyperlinks.each do |link| %>
+                <% participant.hyperlinks_array.each do |link| %>
 	    			<a href="<%= link %>" target="_blank"><%= link %></a><br/>
 	    		<% end %>
 	    	<% end %>
     		<!--Display submitted files-->
  			<% @participants.each do |participant| %>
```

Comment:
Again why @participants.each is still needed? Cant we do @participants.first..submitted_files and save a loop?